packages/libraries/cli/examples/federation.products.graphql

```diff
@@ -1,60 +1,74 @@
-enum CURRENCY_CODE {
```

Comment:
I assume the prior version was for Federation 1? But I couldn't get the two example schemas working locally. I've opted to update them to Federation 2. Let me know if I should make a new version instead.